app/assets/stylesheets/pages/_charts.scss

```diff
@@ -429,3 +429,11 @@
   margin-bottom: 20px;
 }
 
+.report_comment {
+  white-space: pre-wrap;
+}
+
+.recommendation_comment {
```

Comment:
since the two classes are using the same styling, I think you can just combine them like:
```
.report_comment, .recommendation_comment {
  white-space: pre-wrap;
}
```